src/__tests__/createFieldArrayProps.spec.js

```diff
@@ -278,7 +278,7 @@ const describeCreateFieldProps = (name, structure, expect) => {
       expect(result.fields.map).toBeA('function')
       expect(callback).toNotHaveBeenCalled()
       const mapResult = result.fields.map(callback)
-      expect(size(mapResult), 3)
```

Comment:
Should we just remove these? For some reason `size(mapResult)` returns `undefined`.

`Error: Expected undefined to be 3`

https://travis-ci.org/erikras/redux-form/jobs/273908404